project/plugins.sbt

```diff
@@ -1,7 +1,11 @@
+addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.5")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.1")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.1")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.8.1")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.7")
-addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.5")
+
+libraryDependencies ++= Seq(
+  "com.github.os72" % "protoc-jar" % "3.11.1"
```

Comment:
We only need the `protoc` compiler binary from this jar. It's usually a few versions behind but fine.

Reply:
what does this do? I read https://www.scala-sbt.org/1.x/docs/Plugins.html and I think, from that, that this allows us to use protoc-jar stuff in build.sbt - which doesn't seem necessary, since we compile the protos in each module and not in the build? I think I am missing something

Reply:
Protobuf compiler, i.e. `.proto` -> `.java`, is native binary (C++) and not part of `sbt-protobuf`. `sbt-protobuf` calls whatever `protoc` in `PATH` by default.
`protoc-jar` packages pre-built binaries for multiple platforms.